eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumWatcher.scala

```diff
@@ -231,7 +232,7 @@ object ElectrumWatcher extends App {
     def receive = {
       case ElectrumClient.ElectrumReady(_, _, _) =>
         log.info(s"starting watcher")
-        context become running(context.actorOf(Props(new ElectrumWatcher(client)), "watcher"))
+        context become running(context.actorOf(Props(new ElectrumWatcher(new AtomicLong(0), client)), "watcher"))
```

Comment:
The first property of `ElectrumReady` is the current height, shouldn't you use that instead of `0`?
Overall I must admit I don't understand how this is going to work. If you're using your own atomic `blockCount` instead of the shared one, the `ElectrumWatcher` will never have the correct value in `this.blockCount.get()` :confused:

Reply:
This was actually unused code, I removed it in 427f4e7.